src/stores/pacbioPoolCreate.js

```diff
@@ -668,6 +648,27 @@ export const usePacbioPoolCreateStore = defineStore('pacbioPoolCreate', {
         //Populate wells
         this.resources.wells = dataToObjectById({ data: wells, includeRelationships: true })
 
+        // Set used_aliquots into a variable
+        const usedAliquots = dataToObjectById({
+          data: aliquots,
+          includeRelationships: true,
+        })
+
+        // Set the used_aliquots
+        Object.values(usedAliquots).forEach((usedAliquot) => {
+          // Creates a request association for the used aliquot based on the source type
+          usedAliquot.request =
+            usedAliquot.source_type == 'Pacbio::Request'
+              ? this.resources.requests[usedAliquot.source_id].id
+              : this.resources.libraries[usedAliquot.source_id].request
+
+          const key = `_${usedAliquot.request}`
```

Comment:
Im not entirely sure on the best way to store these selected aliquots. Currently we are using the aliquot's request but this comes with the issue that if the same request is wanting to be used twice, e.g. in a library and a plate then it will cause issues. We have the same issue if we use source_id instead as multiple plates/tubes/libraries may have the same id.

Reply:
Maybe not using a special id key at all and tracing aliquots back to their well/tube via a mix of the aliquot source_type and its request.